3ds Max/Max2Babylon/Max2Babylon_Package.py

```diff
@@ -1,14 +1,137 @@
+# BEFORE USING THIS SCRIPT, PLEASE MAKE SURE PYTHON AND THE PACKAGES LISTED IN THE README.MD ARE INSTALLED.
+
+# ---------- IMPORTS ----------
 import os
 import zipfile
 import shutil
+import json     # Changelog
+import requests  # Changelog
+import datetime  # Changelog
 
 versions = ['2015', '2017', '2018', '2019']
 projectFolderPrefix = './'
 buildPathPrefix = '/bin/Release/'
 
 packageFolderPrefix = '../'
 Max2BabylonPackagePrefix = 'Max2Babylon-'
-Max2BabylonVersion = '1.4.0'
+
+currentVersionTime = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+lastVersionTime = ''
+
+# ---------- EXPORTER VERSION ----------
+# Retrieved for the "Exporter/BabylonExporter.cs" file
+
+Max2BabylonVersion = ''
+with open('Max2Babylon/Exporter/BabylonExporter.cs', 'r') as file_BabylonExporterCS:
+    lineIndex = 0
+    for line in file_BabylonExporterCS:
+        lineIndex += 1
+        if line.find('exporterVersion = "') != -1:
+            firstOcc = line.find('"')
+            lastOcc = line.find('"', firstOcc + 1)
+            Max2BabylonVersion = line[firstOcc + 1: lastOcc]
+            break
+file_BabylonExporterCS.close()
+# Be sure that the version has been found. Otherwise, raise a Value Error
+if (Max2BabylonVersion == ''):
+    raise ValueError(
+        "Impossible to fine line with 'string exporterVersion = \"' in the BabylonExporter.cs file")
+
+
+# ---------- UPDATING THE CHANGELOG.MD ----------
+
+# Get informations from the repo
+print("Please, enter your Github Access Token.")
+print("See the readme.md for more informations at ## Build package")
+print("Leave blank and press enter if you don't want to fill the Changelog")
+accessToken = str(input("Access Token : "))
+
+def updateChangelog () :
+
+    queryString = """
+    {
+    repository(owner: "BabylonJS", name: "Exporters") {
+        pullRequests(first:25, states:MERGED, labels:"3dsmax", orderBy:{field:UPDATED_AT, direction:DESC}) {
+        nodes {
+            updatedAt,
+            closedAt,
+            number,
+            title,
+            labels(first:10) {
+            edges {
+                node {
+                name
+                }
+            }
+            }
+        }
+        }
+    }
+    }
+    """
+    query = {'query': queryString.replace('\n', ' ')}
+    headers = {'Authorization': 'token ' + accessToken,
+            'Content-Type': 'application/json'}
+
+    response = requests.post(
+        'https://api.github.com/graphql', headers=headers, json=query)
+
+    if (json.loads(response.text)['message']) :

```

Comment:
If the response is data, then it wouldn't include a message, so this would cause us get a KeyError.

Reply:
My bad ! I didn't test correctly